src/oc_erchef/apps/oc_chef_wm/src/oc_chef_action_queue.erl

```diff
@@ -34,3 +43,65 @@ publish(RoutingKey, Data) ->
     AMQPMsg = bunny_util:new_message(Data),
     PersistentData = bunny_util:set_delivery_mode(AMQPMsg, 2),
     ok = bunnyc:publish(?SERVER, RoutingKey, PersistentData).
+
+get_rabbit_management_pool_setting() ->
```

Comment:
@stevendanna what do you think of me putting the management code in a different module. I feel like the publish code is tightly tied to analytics